packages/webapp/src/stories/Animals/Details/mockData.js

```diff
@@ -0,0 +1,72 @@
+/*
+ *  Copyright 2024 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+export const typeOptions = [
+  { value: 'default_1', label: 'Cattle' },
+  { value: 'default_2', label: 'Pig' },
+  { value: 'default_3', label: 'Chicken' },
+  { value: 'custom_1', label: 'Sheep' },
+];
+
+export const breedOptions = [
+  { value: '1', label: 'Angus' },
+  { value: '2', label: 'Cobb 5' },
+];
+
+export const sexOptions = [
+  { value: 0, label: `I don't know` },
```

Comment:
It's not a question for this PR, but regarding my backend ticket... 'I don't know' is just a frontend implementation of `null`, right? We don't have to add a key in the `animal_sexes` table for it?

Reply:
> If we didn't have the key, "sex" would be "I don't know" in the read-only view when the user didn't select anything.

@SayakaOno yes, I think I prefer this!!

I think I also just saw something like this in another place on the AddAnimal flow -- maybe an "I don't know" breed? -- and I could have _sworn_ we scrapped the idea of an `UNKNOWN` breed key at one point in discussion 😅 At least for sure they didn't make it into the tables.

Let's revisit after soil amendment!